site/src/component/RecentOfferings/RecentOfferings.scss

```diff
@@ -0,0 +1,34 @@
+.recent-offerings-container {
+  overflow-x: auto;
+  max-width: 100%;
+}
+.recent-offerings-table {
+  border-spacing: 10px 5px;
+  font-size: 16px;
+  min-width: 400px;
+}
+
+.recent-offerings-table td,
+th {
+  min-width: 50px;
+  padding-left: 10px;
+  padding-top: 3px;
+  padding-bottom: 3px;
+}
```

Comment:
The emojis and check marks are not the same width, so you should center these.

Additionally, the style rule should be `.recent-offerings-table :is(td, th)` because otherwise you're saying `.recent-offerings-table td` and all `th` elements